inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/util/Config.java

```diff
@@ -73,37 +72,24 @@ private void initIP() {
     }
 
     private void initDockerId() {
-        BufferedReader in = null;
         try {
```

Comment:
Suggest using the try-with-resource to handle the exception and finally scenario.